src/main/java/org/mitre/bonnie/cqlTranslationServer/TranslationResource.java

```diff
@@ -84,11 +88,16 @@ public Response cqlPackageToElmPackage(FormDataMultiPart pkg,
     }
   }
   
-  private CqlTranslator getTranslator(File cql) {
+  private CqlTranslator getTranslator(File cql, boolean disablePromotion) {
```

Comment:
Did you consider overloading `getTranslator` to have the previous behavior with 1 argument, but the new behavior if a second argument is supplied? I'm not sure if it's the best way to go, but it seems like it might be a good way to do things here.

Reply:
If it's not too much trouble, I agree w/ Luke.  You can create a one-arg `getTranslator` function that just calls the two-arg version with `false` -- then you can revert the changes in lines 48 and 58.  That said, it's not a super big deal.